app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module CommentsHelper
+  def comment_html_path(comment)
```

Comment:
so i actually preferred your previous approach of defining this on the class itself
"Helper" classes are meant to be view helpers, but the comment url is useful everywhere, not just in a view context. Plus, you won't have to import it every time.